contracts/upgradeable_contracts/amb_native_to_erc20/ForeignAMBNativeToErc20.sol

```diff
@@ -196,15 +196,15 @@ contract ForeignAMBNativeToErc20 is BasicAMBNativeToErc20, ReentrancyGuard, Base
     * @dev Distributes the provided amount of fees.
     * @param _feeManager address of the fee manager contract
     * @param _fee total amount to be distributed to the list of reward accounts.
-    * @param _txHash the transaction hash associated that generated fee distribution
+    * @param _messageId id of the message that generated fee distribution
     */
-    function distributeFee(IMediatorFeeManager _feeManager, uint256 _fee, bytes32 _txHash) internal {
+    function distributeFee(IMediatorFeeManager _feeManager, uint256 _fee, bytes32 _messageId) internal {
         // Right now, AMB bridge supports only one message per transaction.
         // The receivers of the fee could try to send back the fees through the mediator,
         // so here we add a lock to limit the number of messages that the mediator can send to the bridge,
         // allowing a maximum of 1 message
         enableMessagesRestriction();
```

Comment:
Now that AMB supports multiple messages per transaction, I think this restriction `enableMessagesRestriction` is no longer needed, right @akolotov ?